### PR TITLE
Support for avatar objects + laser beams

### PR DESCRIPTION
**Pull request by benptc**
- Optimizes camera to work with an avatar object
- Adds SHIFT keyboard shortcut to trigger laser beam while held down
- simplifies multiclientUI icosahedron into a smaller cube (for now – still could use improvement but looking for something more subtle) 

Requires https://github.com/ptcrealitylab/vuforia-spatial-toolbox-userinterface/pull/243

### CODE REVIEW

File: content_scripts/desktopStats.js [hunk 1]
```diff
@@ -29,20 +29,29 @@ createNameSpace('realityEditor.device.desktopStats');
     function initService() {
         if (!realityEditor.device.desktopAdapter.isDesktop()) { return; }
 
-        document.body.appendChild(stats.dom);
+        setTimeout(() => {
```
Comment (hobinjk-ptc) [inline content_scripts/desktopStats.js:32]: Is there a way to remove the race condition this is working around?

File: content_scripts/VirtualCamera.js [hunk 2]
```diff
@@ -578,7 +578,12 @@ import * as THREE from '../../thirdPartyCode/three/three.module.js';
             // tween the matrix every frame to animate it to the new position
             // let cameraNode = realityEditor.sceneGraph.getSceneNodeById('CAMERA');
             let currentCameraMatrix = realityEditor.gui.ar.utilities.copyMatrix(this.cameraNode.localMatrix);
-            let newCameraMatrix = tweenMatrix(currentCameraMatrix, realityEditor.gui.ar.utilities.invertMatrix(destinationCameraMatrix), 0.3);
+            let destinationCameraMatrix = realityEditor.gui.ar.utilities.invertMatrix(newCameraLookAtMatrix);
+            let totalDifference = sumOfElementDifferences(destinationCameraMatrix, currentCameraMatrix);
+            if (totalDifference < 0.00001) {
+                return; // don't animate the matrix with an infinite level of precision, stop when it gets very close to destination
+            }
+            let newCameraMatrix = tweenMatrix(currentCameraMatrix, destinationCameraMatrix, 0.3);
```
Comment (hobinjk-ptc) [inline content_scripts/VirtualCamera.js:586]: Is this where it would turn off tweening for first person mode? Or is that somewhere with the velocity

Comment (benptc) [reply content_scripts/VirtualCamera.js]: Just realized I forgot to include that feature. Pushed a new commit with those lines, they indeed go here

**Review by hobinjk-ptc — APPROVED** [review]
Excellent, LGTM! Love the more durable menubar approach!

**Comment by benptc** [comment]
Thanks for the reviews!

**Pull request merged**